crypto/fipsmodule/ec/p384.c

```diff
@@ -503,8 +503,22 @@ static int ec_GFp_nistp384_cmp_x_coordinate(const EC_GROUP *group,
   return 0;
 }
 
+// ----------------------------------------------------------------------------
 //                    SCALAR MULTIPLICATION OPERATIONS
 // ----------------------------------------------------------------------------
+//
+// The method for computing scalar products in functions:
+//   - |ec_GFp_nistp384_point_mul|,
+//   - |ec_GFp_nistp384_point_mul_base|,
+//   - |ec_GFp_nistp384_point_mul_public|,
+// is taken from ECCKiila project (https://arxiv.org/abs/2007.11481).
```

Comment:
changed taken to adapted. About the rest:
- the loop processing is different in mul_base --> done
- we compute the table in our own script --> done
- we define the window bits together rather than bit by bit --> this is the same as in ECCKiila
- we re-used the recoding we had for mul_public --> added a comment about mul_public window size difference, but the recoding is the same as in ECCKiila so I don't think we have to comment on it.